features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/receipt/TimelineItemReadReceiptView.kt

```diff
@@ -56,7 +57,11 @@ fun TimelineItemReadReceiptView(
 ) {
     if (state.receipts.isNotEmpty()) {
         if (renderReadReceipts) {
-            ReadReceiptsRow(modifier = modifier) {
+            ReadReceiptsRow(
+                modifier = modifier.clearAndSetSemantics {
+                    invisibleToUser()
```

Comment:
Is it OK to hide RR when screen reader is off? Maybe we could summarize them by "Seen by 2 people", and for DM it is still a valuable information to speak out

Reply:
I'm not sure if I'd like to hear the same 'seen by X people' part for every recent message. Also, what about state events, should we also mention they were seen by someone?

It may be more important for DM, that's true, but we'd have to think about how to represent this, maybe with some 'This is the last message seen by X' description.

Reply:
Can we keep it like this and iterate on it once we have design/product feedback?

Reply:
Yes, OK